### ISSUE TITLE
WebP files don't show up in link/image path completion

### ISSUE THREAD

**Issue by alexilyaev** [yzhang-gh/vscode-markdown#934]
## Problem

WebP files don't show up in link/image path completion.

![image](https://user-images.githubusercontent.com/480469/115144783-3ebb6700-a057-11eb-9fe2-64403807f180.png)

### Env

Markdown All in One `v3.4.0`

**VS Code:**

```
Version: 1.55.2
Commit: 3c4e3df9e89829dce27b7b5c24508306b151f30d
Date: 2021-04-13T09:36:32.643Z (5 days ago)
Electron: 11.3.0
Chrome: 87.0.4280.141
Node.js: 12.18.3
V8: 8.7.220.31-electron.0
OS: Darwin x64 19.6.0
```

## How to reproduce

- Put a bunch of images in a folder (`png`, `jpg` and `webp`)
- Start writing a link or image in Markdown until the autocomplete pops up
- Should see `webp` files, but they don't show up

## Error message in the console

No errors in DevTools console.

**Comment by yzhang-gh** [comment]
https://github.com/yzhang-gh/vscode-markdown/blob/908d7ba5465a203e4299f346c179211d992ef468/src/completion.ts#L453

The code looks quite right so there shouldn't be this issue. Perhaps you need to recheck it with another folder.

**Comment by Lemmingh** [comment]
Oops, I just realized that we haven't shipped version 3.5.0 yet, that is, the changes we made in the past 5 months are not in the stable release.

9d4cef7775317ac2b2edb3019e09e145c3923610

**Comment by yzhang-gh** [comment]
@Lemmingh Me too 😅

@alexilyaev You can use the [dev build](https://github.com/yzhang-gh/vscode-markdown#latest-development-build) which supports `webp`.

**Comment by yzhang-gh** [comment]
released in v3.4.2
